### PR TITLE
Change DJ Caption for file

### PR DESCRIPTION
**Pull request by braclaver**
Summary: This adds a caption for the teacher file

# Things accomplished

Add a caption
Summary of change 2
Summary of change 3

cc @amandeepjaswal 
cc @linak97

### CODE REVIEW

File: _posts/0000-01-26-TerHano.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Caption here
```
Comment (amandeepjaswal) [inline _posts/0000-01-26-TerHano.md:6]: You forgot to add caption here.

**Review by amandeepjaswal — CHANGES_REQUESTED** [review]
good work !

File: _posts/0000-01-26-TerHano.md (hunk 1 above)
Comment (braclaver) [reply _posts/0000-01-26-TerHano.md:6]: Ok Sir, got it !

File: _posts/0000-01-26-TerHano.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
```
Comment (amandeepjaswal) [inline _posts/0000-01-26-TerHano.md:4]:
```suggestion
--- hahaha
```

**Review by amandeepjaswal — CHANGES_REQUESTED** [review]
Awsome!

**Review by amandeepjaswal — APPROVED** [review]

**Pull request merged**